src/utils/constants.js

```diff
@@ -111,6 +111,8 @@ export const VALIDATION_MESSAGES = {
   START_TIME: 'Please enter a valid date later than now',
   END_TIME: 'Please enter a valid date later than start time',
   MULTIPLE_TIERS_START_TIME: 'Please enter a valid date not less than the end time of the previous tier',
+  EDITED_END_TIME: 'Please enter a valid date later than start time and previous than start time of next tier',
+  EDITED_START_TIME: 'Please enter a valid date later than now, less than end time and less than the end time of the previous tier',
```

Comment:
Should be `and later than the end time of the previous tier`.

Reply:
done